VideoLocker/src/main/java/org/edx/mobile/view/custom/popup/menu/MenuPopupHelper.java

```diff
@@ -148,14 +148,22 @@ public MenuPopupHelper(Context context, MenuBuilder menu, View anchorView,
             mPopupPaddingTop = popupPadding;
             mPopupPaddingBottom = popupPadding;
         } else {
-            mPopupPaddingStart = a.getDimensionPixelSize(
-                    R.styleable.PopupMenu_android_paddingStart,
-                    a.getDimensionPixelOffset(
-                            R.styleable.PopupMenu_android_paddingLeft, 0));
-            mPopupPaddingEnd = a.getDimensionPixelSize(
-                    R.styleable.PopupMenu_android_paddingEnd,
-                    a.getDimensionPixelOffset(
-                            R.styleable.PopupMenu_android_paddingRight, 0));
+            int paddingStart = Integer.MIN_VALUE;
+            int paddingEnd = Integer.MIN_VALUE;
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
+                paddingStart = a.getDimensionPixelSize(
+                        R.styleable.PopupMenu_android_paddingStart, Integer.MIN_VALUE);
+                paddingEnd = a.getDimensionPixelSize(
+                        R.styleable.PopupMenu_android_paddingEnd, Integer.MIN_VALUE);
+            }
+            if (paddingStart == Integer.MIN_VALUE && paddingEnd == Integer.MIN_VALUE) {
```

Comment:
This condition seems to be erroneous, because, if one of the attribute among `PopupMenu_android_paddingStart` and `PopupMenu_android_paddingEnd` is undefined on SDK level `>= Build.VERSION_CODES.JELLY_BEAN_MR1` its respective local variable will have `Integer.MIN_VALUE` as its value, which ideally should have `0` has its value.